features/text-overflow.yml

```diff
@@ -3,8 +3,12 @@ description: The `text-overflow` CSS property sets how hidden overflow content a
 spec: https://drafts.csswg.org/css-overflow-4/#text-overflow
 group: css
 caniuse: text-overflow
+status:
+  compute_from: css.properties.text-overflow
 compat_features:
   - css.properties.text-overflow
   - css.properties.text-overflow.clip
   - css.properties.text-overflow.ellipsis
   - svg.global_attributes.text-overflow
+  - css.properties.text-overflow.string
```

Comment:
This particular key seems interesting as it allows you to define your own ellipsis (if I understand correctly).
I'm tempted to make a separate feature for this. What do you think?

Reply:
I added a `Custom ellipsis` feature.